test/indexer/test_hca_indexer.py

```diff
@@ -1027,6 +1031,41 @@ def test_accessions_fields(self):
             self.assertEqual(['E-AAAA-00'], project['array_express_accessions'])
             self.assertEqual(['PRJNA000000'], project['insdc_study_accessions'])
 
+    def test_no_cell_count_contributions(self):
+        def assert_cell_suspension(expected_result: Dict, hits):
```

Comment:
Dict is generic

Reply:
opted to omit, method is only used within this testcase.